### DISCUSSION TITLE
Do Not Push

### DISCUSSION THREAD

**Discussion by joeromine** [Vets-Who-Code/march2021#59] [Ideas]
I think it would be cool if the 'do not push' button would open in a modal. 

Also, if it played a shrieking sound . . . https://www.youtube.com/watch?v=yDRYEh6iK8A 

Any takers?

**Comment by dtetreau251** [comment]
Modal?

**Comment by dtetreau251** [comment]
Ok you got my attention with the modal. Didn’t even know that term.  I’ll take that on! 


Sent from Yahoo Mail for iPhone


On Wednesday, March 10, 2021, 10:12 PM, David Tetreau ***@***.***> wrote:

Modal? 

Sent from my iPhone

On Mar 10, 2021, at 10:05 PM, Stephan Lamoureux ***@***.***> wrote:



﻿


I never pair coded before or created a modal, could knock out both 🤔

**Comment by joeromine** [comment]
https://getbootstrap.com/docs/4.0/components/modal/

**Comment by joeromine** [comment]
https://codepen.io/NielsVoogt/pen/XWjPdjO

**Comment by joeromine** [comment]
Something like this
https://codepen.io/NielsVoogt/pen/XWjPdjO

**Reply by dtetreau251** [reply]
i made this an issue
